tests/everflow/everflow_test_utilities.py

```diff
@@ -406,17 +609,24 @@ def setup_mirror_session(self, duthosts, rand_one_dut_hostname, config_method):
         Yields:
             dict: Information about the mirror session configuration.
         """
-        duthost = duthosts[rand_one_dut_hostname]
-        session_info = BaseEverflowTest.mirror_session_info("test_session_1", duthost.facts["asic_type"])
+        if setup_info['topo'] == 't2':
+            duthost_list = [setup_info[DOWN_STREAM]['everflow_dut']]
+            if setup_info[UP_STREAM]['everflow_dut'] != setup_info[DOWN_STREAM]['everflow_dut']:
+                duthost_list.append(setup_info[UP_STREAM]['everflow_dut'])
+        else:
+            duthost_list = [duthosts[rand_one_dut_hostname]]
 
-        BaseEverflowTest.apply_mirror_config(duthost, session_info, config_method)
+        for duthost in duthost_list:
+            session_info = BaseEverflowTest.mirror_session_info("test_session_1", duthost.facts["asic_type"])
+            BaseEverflowTest.apply_mirror_config(duthost, session_info, config_method)
 
         yield session_info
 
-        BaseEverflowTest.remove_mirror_config(duthost, session_info["session_name"], config_method)
+        for duthost in duthost_list:
+            BaseEverflowTest.remove_mirror_config(duthost, session_info["session_name"], config_method)
 
     @pytest.fixture(scope="class")
-    def policer_mirror_session(self, duthosts, rand_one_dut_hostname, config_method):
+    def policer_mirror_session(self, duthosts, enum_rand_one_per_hwsku_frontend_hostname, config_method, setup_info):
```

Comment:
@judyjoseph  this name "enum_rand_one_per_hwsku_frontend_hostname" sounds like for multi asic or T2 only, could we  use a common name like "rand_one_dut_hostname"?